testing/scenario_app/compile_android_jit.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
2020?

Reply:
I think the license banner consistently uses 2013. https://github.com/flutter/engine/search?p=5&q=Copyright+2013&unscoped_q=Copyright+2013